google-cloud-spanner/src/main/java/com/google/cloud/spanner/connection/DdlClient.java

```diff
@@ -35,11 +36,13 @@
  */
 class DdlClient {
   private final DatabaseAdminClient dbAdminClient;
+  private final String projectId;
   private final String instanceId;
   private final String databaseName;
 
   static class Builder {
     private DatabaseAdminClient dbAdminClient;
+    private String projectId;
```

Comment:
nit: Instead of adding this field, could we change the individual fields `projectId`, `instanceId`, `databaseName` into one field of type `DatabaseId`?

Reply:
sure, will do this along with the previous comment as a seperate PR.

Reply:
This comment will be addressed in PR https://github.com/googleapis/java-spanner/pull/3126